selfdrive/car/honda/values.py

```diff
@@ -1302,4 +1302,5 @@ class CAR:
 
 HONDA_NIDEC_ALT_PCM_ACCEL = set([CAR.ODYSSEY])
 HONDA_BOSCH = set([CAR.ACCORD, CAR.ACCORDH, CAR.CIVIC_BOSCH, CAR.CIVIC_BOSCH_DIESEL, CAR.CRV_5G, CAR.CRV_HYBRID, CAR.INSIGHT, CAR.ACURA_RDX_3G, CAR.HONDA_E])
-HONDA_BOSCH_ALT_BRAKE_SIGNAL = set([CAR.ACCORD, CAR.CRV_5G, CAR.ACURA_RDX_3G])
\ No newline at end of file
+HONDA_BOSCH_ALT_BRAKE_SIGNAL = set([CAR.ACCORD, CAR.CRV_5G, CAR.ACURA_RDX_3G])
+HONDA_NO_WHEELS_MOVING_SIGNAL = {CAR.ACCORD, CAR.ACCORDH, CAR.CIVIC_BOSCH, CAR.CIVIC_BOSCH_DIESEL, CAR.CRV_HYBRID, CAR.INSIGHT, CAR.ACURA_RDX_3G, CAR.HONDA_E, CAR.ODYSSEY_CHN}
```

Comment:
Instead of these lists, somehow I'd like to represent the setting in car params, so that the configuration of each model can be centralized in `interface.py`.

something like:
```python
  class HondaDetails(NamedTuple):
    nidec_alt_pcm_accel = False
    is_bosch = False
    ...

  def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):
    ret = CarInterfaceBase.get_std_params(candidate, fingerprint)
    ret.carName = "honda"
    ret.details = HondaDetails()

    ...
    elif candidate == CAR.ODYSSEY:
      ret.details.nidec_alt_pcm_accel = True
      ...
      

```

Reply:
the new `flags` field in CarParams would allow this, but I'll try it in a separate PR